solr/core/src/java/org/apache/solr/security/RuleBasedAuthorizationPlugin.java

```diff
@@ -41,17 +40,11 @@ public void init(Map<String, Object> initInfo) {
       String roleName = e.getKey();
       usersVsRoles.put(roleName, Permission.readValueAsSet(map, roleName));
     }
-    useShortName =
-        Boolean.parseBoolean(initInfo.getOrDefault("useShortName", Boolean.FALSE).toString());
```

Comment:
what's this about?

Reply:
we have this "useShortName" feature, but it turns out it's ONLY ever supported by Kerberos.   So it is dead code/unused feature with that...

Reply:
Yea the short name in Kerberos is without the realm. So if the principal was `USERNAME@REALM` it would just be `USERNAME` or if the principal was `USERNAME/HOST@REALM` it would just be `USERNAME` and is a Kerberos construct to break down the whole principal